tests/__helpers__/jest-config.ts

```diff
@@ -0,0 +1,210 @@
+import { TsJestConfig } from '../../dist/types';
+
+const { resolve } = require.requireActual('path');
+
+function createJestConfig(
```

Comment:
I think I see what you're trying to do with this, but after looking at the examples where this has actually been used, and how difficult this code is to fathom (took me a while) I don't think this level of abstraction is worth the cost of comprehending it. It either needs to go, or be implemented in a much cleaner way. I'd suggest we kill it for now, and then try to work it into another PR

Reply:
OOooooohhh yeah, that is where I forgot a todo! I wanted to cleanup that crap, you can say it, it's crap! I'll see what I can do to remove it but keep the tests of course